airbyte-api/src/main/openapi/config.yaml

```diff
@@ -359,6 +359,145 @@ paths:
           $ref: "#/components/responses/NotFoundResponse"
         "422":
           $ref: "#/components/responses/InvalidInputResponse"
+  /v1/source_definitions/list_private:
```

Comment:
and this endpoint is used for administrative purposes only? maybe worth saying that in the description.

Reply:
ah yes it is, good point